units/src/amount/signed.rs

```diff
@@ -150,12 +204,12 @@ impl SignedAmount {
     #[cfg(feature = "alloc")]
     pub fn to_btc(self) -> f64 { self.to_float_in(Denomination::Bitcoin) }
 
-    /// Convert this [`SignedAmount`] in floating-point notation with a given
-    /// denomination.
+    /// Convert this [`SignedAmount`] in floating-point notation in the given
+    /// [`Denomination`].
```

Comment:
```suggestion
    /// Converts this [`SignedAmount`] in floating-point notation in the given [`Denomination`].
```

100 column width for rusdocs in this repo (unless breaking the line shorter makes it look noticeably, or even subjectively, better).

Reply:
Just FYI, this is a repo policy we have in place. It is not a Rust-wide thing.

Reply:
And no there is no reason other than there has to be some policy and the exact value requires weeks of bike-shedding and inevitably someone doesn't like the final value. But to quote @dpc "hell is other people" :)

Reply:
Oh, and just as a general open source contributing thing; fixing column width has a tendency to annoy people because its kind of seen as code churn but @apoelstra and I will typically be ok to review it - especially in the leaf crates where, at least I personally, think it is a definite should-be-fixed-before 1.0 because it looks more professional if the docs are super clean.

Reply:
Done